visualization/app/codeCharta/util/scenarioHelper.ts

```diff
@@ -1,41 +1,254 @@
 "use strict"
-import { MetricData, RecursivePartial, Settings } from "../codeCharta.model"
+import {
+	AppSettings,
+	CCLocalStorage,
+	DynamicSettings,
+	MetricData,
+	RecursivePartial,
+	ScenarioAsSettings,
+	Scenario,
+	Settings
+} from "../codeCharta.model"
 import { convertToVectors } from "./settingsHelper"
-
-export interface Scenario {
-	name: string
-	settings: RecursivePartial<Settings>
-}
+import { AddScenarioContent, ScenarioMetricType } from "../ui/dialog/dialog.addScenarioSettings.component"
+import { ScenarioItem } from "../ui/scenarioDropDown/scenarioDropDown.component"
 
 export class ScenarioHelper {
+	private static readonly CC_LOCAL_STORAGE_VERSION = "1.0.0"
 	//TODO: Move Scenarios to Redux Store
-	private static scenarios: Scenario[] = ScenarioHelper.importScenarios(require("../assets/scenarios.json"))
+	private static scenarioList: RecursivePartial<Scenario>[] = ScenarioHelper.loadScenarios()
```

Comment:
It would be better to use a Map instead of an array for all scenarios. That way they immediately accessible by their name and there is no need to first find the corresponding scenario inside of an array.